ci/jenkins/jobs/macros.yaml

```diff
@@ -275,3 +275,30 @@
           set -e 
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
           ./ci/jenkins/test-mc.sh --testcase e2e --registry ${DOCKER_REGISTRY} --mc-gateway --codecov-token "${CODECOV_TOKEN}" --coverage --kind
+            
+- builder:
+    name: builder-e2e-jumper
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -e
+          DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          ./ci/jenkins/test.sh --testcase e2e --registry ${DOCKER_REGISTRY} --testbed-type jumper
+
+- builder:
+    name: builder-conformance-jumper
```

Comment:
Can we add `conformance_type` like builder-conformance here? Then builder-networkpolicy-jumper is not needed.

Reply:
`conformance_type` added.